docs/Fortigate-Events-Legacy-Configuration.md

```diff
@@ -0,0 +1,71 @@
+# Configuration for collection of Fortinet Fortigate logs (Preview)
+
+**This feature is part of the OMS Security and Audit solution. The solution needs to be enabled before the events can be collected.**
+
+Collection of the following 3rd party security log types is supported:
+- **Fortinet Fortigate logs FortiOS < 5.6**
+
+Fortinet Fortigate can send events in CEF starting from [FortiOS 5.6](http://help.fortinet.com/fos50hlp/56/Content/FortiOS/fortigate-whats-new/Top-LogReport-cef.htm?Highlight=syslog%20CEF), if it is not possibile to update the appliances to the new OS, this solution can be used to translate legacy Fortigate Logs in CEF and ingest them into Log Analytics.
+
+## Caveats
+
+Fortigate log parsing can have an impact on CPU utilization by the OMS agent, the solution has been tested on a dual core Virtual Machine ingesting about 40 events per second causing an increase in CPU usage of about 5%.
+
+## Configuration summary
+1. Install and onboard the OMS Agent for Linux
+2. Configure Syslog forwarding to send the required logs to the agent on UDP port 25227. This must be a dedicated syslog to be able to differentiatte >Fortigate logs from other log types
+3. Place the agent configuration [file][1] on the agent machine in ```/etc/opt/microsoft/omsagent/<workspace id>/conf/omsagent.d/```
+4. Restart the syslog daemon and the OMS agent
```

Comment:
please provide exact cmdline that user needs to run

Reply:
I cannot since it depends in which workspace the user wants to enable to solution in, same pattern as the security events docs https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/docs/Security-Events-Preview-Configuration.md